README.rst

```diff
@@ -7,50 +7,98 @@ Ptychodus
 * `ptychopy`_ is under development
 * `PtychoNN`_ is under development
 
-Installation
-------------
+Standard Installation
+---------------------
 
-* To install `ptychodus`, install `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ then:
+1. Install `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_.
+
+2. Install ptychodus.
+
+   * To install `ptychodus` with the GUI and no optional packages:
+
+     .. code-block:: shell
+
+           $ conda create -c conda-forge -n ptychodus ptychodus
+
+   * To install `ptychodus` with the GUI and all optional packages:
+
+     .. code-block:: shell
+
+           $ conda create -c conda-forge -n ptychodus ptychodus-all
+
+   * To install `ptychodus` without the GUI or optional packages:
+
+     .. code-block:: shell
+
+           $ conda create -c conda-forge -n ptychodus ptychodus-core
+
+3. Activate the `ptychodus` conda environment to run ptychodus.
+
+   .. code-block:: shell
+
+       $ conda activate ptychodus
+       $ ptychodus -h
+       usage: ptychodus [-h] [-b] [-d] [-f PREFIX] [-p PORT] [-s SETTINGS] [-v]
+
+       ptychodus is a ptychography analysis front-end
+
+       optional arguments:
+         -h, --help            show this help message and exit
+         -b, --batch           run reconstruction non-interactively
+         -d, --dev             run in developer mode
+         -f PREFIX, --file-prefix PREFIX
+                               replace file path prefix
+         -p PORT, --port PORT  remote process communication port number
+         -s SETTINGS, --settings SETTINGS
+                               use settings from file
+         -v, --version         show program's version number and exit
+       $ ptychodus
+
+
+Developer Installation
+----------------------
+
+* For a developer installation:
 
 .. code-block:: shell
 
-    $ git clone https://github.com/AdvancedPhotonSource/ptychodus.git
-    $ conda create -c conda-forge -n ptychodus --file ptychodus/requirements-dev.txt
-    $ conda activate ptychodus
-    $ pip install -e ./ptychodus
+   $ git clone https://github.com/AdvancedPhotonSource/ptychodus.git
+   $ conda create -c conda-forge -n ptychodus --file ptychodus/requirements-dev.txt
+   $ conda activate ptychodus
+   $ pip install -e ./ptychodus
 
 * To install the `tike` backend:
 
 .. code-block:: shell
 
-    $ conda install -n ptychodus -c conda-forge tike
+   $ conda install -n ptychodus -c conda-forge tike
 
 * To install the `PtychoNN` backend:
 
 .. code-block:: shell
 
-    $ conda install -n ptychodus -c conda-forge pytorch
+   $ conda install -n ptychodus -c conda-forge pytorch
```

Comment:
This doesn't actually work yet. Maybe note that?

Reply:
It's mentioned at the top of the README but not here. Hopefully we can knock out the PtychoNN quickly so this is no longer an issue, but I'm open to whatever you think is best until then.